be/src/exec/parquet_writer.cpp

```diff
@@ -49,9 +54,12 @@ Status RollingAsyncParquetWriter::init_rolling_writer(const TableInfo& tableInfo
     return Status::OK();
 }
 
+// prepend fragment instance id to a file name so we can determine which files were written by which fragment instance or be.
+// and we can also know how many files each instance and each driver has written according to file_counts and driver_id mark.
 std::string RollingAsyncParquetWriter::_new_file_location() {
     _file_cnt += 1;
-    _outfile_location = _partition_location + fmt::format("{}_{}.parquet", _file_cnt, generate_uuid_string());
+    _outfile_location = _partition_location + fmt::format("{}_{}_{}.parquet", print_id(_state->fragment_instance_id()),
```

Comment:
If not using uuid, this may lead to data files overwritten. (e.g. two tables use same table/partition location)